src/components/RecommendationTool.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import RecommendationHeader from './RecommendationHeader';
+import PropTypes from "prop-types";
+import DropdownSelect from './dropdown';
+
+const root = {
+    marginTop: 125
+}
+
+const descriptionText = {
+    textAlign:"center", 
+    marginTop: 25,
+    marginBottom: 25
+}
+
+const helpText = {
+    marginTop: -15,
+    marginBottom: 25,
+    textAlign: "center",
+    fontStyle: "italic"
+}
+
+const inputRoot = {
+    margin: "auto",
+    width: "max-content"
+}
+
+const nextButton = {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    "position": "absolute",
+    "width": "64px",
+    "height": "39px",
+    "right": "40px",
+    "bottom": "30px",
+    "background": "#3DBEFF",
+    "borderRadius": "100px",
+    "cursor": "pointer"
+}
+
+const previousButton = {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    "position": "absolute",
+    "width": "94px",
+    "height": "39px",
+    "left": "40px",
+    "bottom": "30px",
+    "background": "#3DBEFF",
+    "borderRadius": "100px",
+    "cursor": "pointer"
+}
+
+const buttonText = {
+    "fontFamily": "Roboto",
+    "fontStyle": "normal",
+    "fontWeight": "bold",
+    "fontSize": "16px",
+    "lineHeight": "19px",
+    "color": "#FFFFFF"
+}
+
+const labelRoot = {
+    display: "block"
+}
+
+const RecommendationTool = (props) => {
+    // eslint-disable-next-line no-unused-vars
+    const [currentStage, setCurrentStage] = useState(0);
+    // eslint-disable-next-line no-unused-vars
+    const [formState, setFormState] = useState({});
+    return <>
+        <RecommendationHeader currentStage={currentStage} stages={props.data.map(x => x.title)}/>
+        <div style={root}>
+            <p style={descriptionText}>{props.data[currentStage].description}</p>
+            {props.data[currentStage].questions.map(question => {
+                let inputComponent = <></>
+                switch (question.type) {
+                    case "multiple_choice":
+                        inputComponent = <>{question.answers.map(answer => <><label key={answer} style={labelRoot}><input type="radio"/>{answer}</label><br/></>)}</>;
```

Comment:
I get a `Warning: Each child in a list should have a unique "key" prop.` on this line.